test/unittests/parser_test.cpp

```diff
@@ -9,6 +9,14 @@ static const auto functype_void_to_void = from_hex("600000");
 static const auto functype_i32i64_to_i32 = from_hex("60027f7e017f");
 static const auto functype_i32_to_void = from_hex("60017f00");
 
+namespace
+{
+bytes push_section(uint8_t id, const bytes& section)
+{
+    return bytes{} + id + uint8_t(section.size()) + section;
```

Comment:
```suggestion
    return bytes{id} + uint8_t(section.size()) + section;
```
I don't like that in previous version there are `uint8_t` connected with `+`.
Also add assert that `section.size() < 0x80`.

Reply:
`ASSERT_TRUE` doesn't work in the helper. Any suggestions?

Reply:
For that to work you would need to create GTest fixture and implement `make_section` as a method of it. https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests